brambling/management/commands/send_daily_emails.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import traceback
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from brambling.mail import DailyDigestMailer
+from brambling.models import Person
+
+
+class Command(BaseCommand):
+    def get_recipients(self):
+        return Person.objects.filter(
+            notify_new_purchases=Person.NOTIFY_DAILY,
```

Comment:
Mm. Though that could result in someone only getting an email every other day. Maybe a compromise like 20 hours?